innovation.game.php

```diff
@@ -16487,6 +16492,7 @@ function stInterInteractionStep() {
                     // Put the rest of the cards of the same color back in the score pile (making it easier to isolate the selectable cards for future interactions)
                     $revealed_cards = self::getCardsInLocation($player_id, 'revealed');
                     foreach ($revealed_cards as $card) {
```

Comment:
This bug doesn't actually happen when the card is in a publicly visible location (e.g. revealed or board). So let's remove this line so that we aren't making an unnecessary SQL query.